src/frontend/src/components/Posts/Posts.jsx

```diff
@@ -39,6 +41,16 @@ const Posts = () => {
     }
   );
 
+  const updateMedia = () => {
+    setDesktopView(window.innerWidth > 1450);
```

Comment:
I think we can do this at much narrower views.  We actually have more space if we shift the timeline of posts to the right and drop the right margin.